hrqb/base/task.py

```diff
@@ -1,69 +1,123 @@
 """hrqb.base.task"""
 
+import os
+from abc import abstractmethod
+from typing import Literal
+
 import luigi  # type: ignore[import-untyped]
 import pandas as pd
 
 from hrqb.base import PandasPickleTarget, QuickbaseTableTarget
+from hrqb.config import Config
 from hrqb.utils import today_date
 from hrqb.utils.quickbase import QBClient
 
 
 class HRQBTask(luigi.Task):
     """Base Task class for all HRQB Tasks."""
 
-    path = luigi.Parameter()
-    table_name = luigi.Parameter()
+    pipeline = luigi.Parameter()
```

Comment:
Curious, what kind of values would be set for `HRQBTask.pipeline`? 🤔

Reply:
This will be the pipeline name.  The _next_ PR finally brings all these pieces together.